week03/706. Design HashMap.cpp

```diff
@@ -0,0 +1,65 @@
+// https://leetcode.com/problems/design-hashmap/
+// Time Complexity: O(1)
+// Space Complexity: O(N)
```

Comment:
Time complexity is O(1)? And what happened with the indentation?

Reply:
Actually, it should be O(N) for chaining element. Will fix it.